packages/flutter/lib/src/rendering/shifted_box.dart

```diff
@@ -143,14 +152,13 @@ class RenderPadding extends RenderShiftedBox {
   }
 }
 
+/// Aligns its child box within itself.
+///
+/// It's only useful for children that don't always size to fit their parent.
+/// For example, to align a box at the bottom right, you would pass this box a
```

Comment:
Move the example above the caveat, or it sounds like the example is about the caveat.